### ISSUE TITLE
does not work when added to existing 4.27 project

### ISSUE THREAD

**Issue by unrealengine4u** [truong-bui/AsyncLoadingScreen#51]
When tested in a blank project it is fine but when it is integrated into an existing it does not.

From the testing I have done, it seems as if there is a hang and it does not actually play the movies.

Tests I have done:
- wait for manual stop 
- use project settings from a blank project and set the plugin settings up
- use a blank level in the project along with
- use the marketplace version (project specific and marketplace give the same results)
- use other video files (all mp4 {MPEG-4} )

Is there something else I should test that might five a more specific reproducible issue?

**Comment by rhesa-beg** [comment]
hey, this problem happens to me as well. It works a couple of months ago, I lost track of it but it suddenly doesn't appear anymore. Do you find a solution?

**Comment by unrealengine4u** [comment]
> hey, this problem happens to me as well. It works a couple of months ago, I lost track of it but it suddenly doesn't appear anymore. Do you find a solution?

yes actually, take a look into the plugins you are using, for me, I disabled "GPU Lightmass" and "Common Widgets". I'm not sure which one caused the issue for me but I'm leaning towards "GPU Lightmass".

hope this helps!

**Comment by rhesa-beg** [comment]
> yes actually, take a look into the plugins you are using, for me, I disabled "GPU Lightmass" and "Common Widgets". I'm not sure which one caused the issue for me but I'm leaning towards "GPU Lightmass".
> 
> hope this helps!

Thank you for the lightning-fast reply! I cannot find "Common widgets" and already disabled "GPU Lightmass". How do you end up figuring it out? What did you check to finally find the problem if I may know?

**Comment by unrealengine4u** [comment]
> > yes actually, take a look into the plugins you are using, for me, I disabled "GPU Lightmass" and "Common Widgets". I'm not sure which one caused the issue for me but I'm leaning towards "GPU Lightmass".
> > hope this helps!
> 
> Thank you for the lightning-fast reply! I cannot find "Common widgets" and already disabled "GPU Lightmass". How do you end up figuring it out? What did you check to finally find the problem if I may know?

No worries!

hm, interesting.

it's a bit of a winding story but to sum it up (sort of):

(pre issue creation)
I updated our project to 4.27.x from 4.26.x and it complained about a material issue. I don't remember the name exactly but it was something like "screen" or "move" but don't quote me on that. but also the project started compiling 8k+ shaders. 

(post issue creation)
fast forward a while I decided to debug it again and implement my own method for loading screens (same as how this plugin does it) and found it to be doing all the right stuff but just not displaying the loading screen, so I went through a bunch of stuff until I came to the plugins and kept only the ones I needed and removed most the 'quality of life' editor plugins. after that and another 8k+ shaders being compiled, it worked.

I'm not really sure what direction I would go next other than to just double-check anything and everything you can.

**Comment by rhesa-beg** [comment]
I will do the same. I appreciate it very much. You shed some ray of hope today!

**Comment by unrealengine4u** [comment]
> I will do the same. I appreciate it very much. You shed some ray of hope today!

Awsome! let me know how you get on

**Comment by rhesa-beg** [comment]
Hello, I finally found it! It is because we use Electra Player, Electra Player Content Decryption module, and Electra Player Utilities.
Gonna ask about this to Mr. Truong.
